src/app/tabbed-menu.js

```diff
@@ -0,0 +1,58 @@
+var $ = require('jquery')
+var yo = require('yo-yo')
+var csjs = require('csjs-inject')
+var styleGuide = require('./style-guide')
+var styles = styleGuide()
+
+module.exports = tabbedMenu
+
+function tabbedMenu (compiler, loadingSpinner, self) {
+
+  var css = csjs`
+    .loadingMsg extends ${styles.warningTextBox} {
+      display: block;
+    }
+  `
+
+  $('#options li').click(function (ev) {
+    var $el = $(this)
+    selectTab($el)
+  })
+
+  // initialize tabbed menu
+  selectTab($('#options .envView'))
+
+  function cb (finish) {
```

Comment:
It is in use... line 23 sets which tab is a default tab (it's not necessary the first one). It was already in the code, so I just changed it to the be a contract tab 

And finish is used to finish spinner after you select new solidity version.